tests/FakeItEasy.Specs/UnconfiguredFakeSpecs.cs

```diff
@@ -200,7 +200,7 @@ public FakedClass()
 
             public virtual string StringProperty { get; set; }
 
-            public string StringPropertyValueDuringConstructorCall { get; }
+            public string? StringPropertyValueDuringConstructorCall { get; }

```

Comment:
The compiler now detects that `StringProperty` might not have a non-null value when we assign it to `StringPropertyValueDuringConstructorCall` in the constructor.